<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1695</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Droid Sans Mono</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Clerk Savior</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QScrollArea" name="imageView">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>802</width>
            <height>644</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="viewPanel">
             <property name="text">
              <string>Double click to open image</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="openImage">
            <property name="text">
             <string>Open image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openPDF">
            <property name="text">
             <string>Open PDF</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextPage">
            <property name="text">
             <string>Next Page</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="batchProcess">
            <property name="text">
             <string>Process file names</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="batchRename">
            <property name="text">
             <string>Rename all files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="models"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="patternBox"/>
       </item>
       <item>
        <widget class="QTableView" name="itemsTable"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
